README.md

```diff
@@ -1,97 +1,39 @@
-# HTML & CSS3 Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# Project 1 Portfolio 
 
-## Set-up GitHub Actions
+> Web design for mobile version
 
-Please do the following **steps in this order**:
+## Built With
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+- HTML, CSS
```

Comment:
- [ ][OPTIONAL] Technologies used list is missing Javascript. it could be helpful in exhausting all technologies . Linters, Javascript